web/server/codechecker_server/api/report_server.py

```diff
@@ -1186,12 +1236,26 @@ def getComments(self, report_id):
                     .order_by(Comment.created_at.desc()) \
                     .all()
 
+                context = webserver_context.get_context()
                 for comment in comments:
+                    message = comment.message
+
+                    if comment.kind == 'system':
```

Comment:
Please convert the Enum value to string for the comparison so it is harder to make mistakes during a renaming.

```suggestion
                    if comment.kind == comment_kind_str(ttypes.CommentKind.SYSTEM):
```